chargeOclock/Source/Other/NetworkAPI.swift

```diff
@@ -19,6 +19,12 @@ extension NetworkAPI {
 	func delete(client: Client, completion: @escaping NetworkCompletion) {
 		performDeleteRequest(to: "clients/\(client.id)", completion: completion)
 	}
+
+	func addClient(name: String, completion: @escaping NetworkCompletion) {
```

Comment:
there will be 2 methods, `create(client: Client)` and `update(client: Client)`
(and the completions, obviously)